server/documents/content.js

```diff
@@ -57,7 +101,7 @@ Meteor.methods({
     });
 
     if (latestContent.version !== args.currentVersion) {
-      return 0;
+      throw new Meteor.Error('version-error', `Step version doesn't match.`);
```

Comment:
Hm, why an error here? I think you should on the client check how many steps were added (return value) and then compare with how many you send? No?